src/main/scala/sigmastate/lang/SigmaSpecializer.scala

```diff
@@ -12,7 +12,7 @@ import sigmastate.lang.Terms.{Apply, Block, Ident, Lambda, Let, Select, ValueOps
 import sigmastate.lang.exceptions.SpecializerException
 import sigmastate.utxo._
 
-class SigmaSpecializer {
+class SigmaSpecializer(val builder: SigmaBuilder = TransformingSigmaBuilder) {
```

Comment:
@greenhat this default argument looks dangerous. Let’s make configuration explicit.

Reply:
@ergomorphic Agreed. Done.